otherlibs/stdune/src/ansi_color.ml

```diff
@@ -494,7 +494,7 @@ let rec tag_handler buf current_styles ppf (styles : Style.t list) pp =
     (Style.escape_sequence_buf buf (current_styles :> Style.t list))
 ;;
 
-let make_printer supports_color ppf =
+let make_printer ~line_break ~supports_color ppf =
```

Comment:
Is there a reason for using a positional argument instead of an `optional` with a `true` as default value?

Reply:
We generally try to avoid optional arguments in the Dune code base as opting users unknowingly into default behaviors is not great. In this case, the outcome of this is fairly limited as `make_printer` is only really used by `print` and `prerr`.